01_draw_functions/01_subfunctions/ControlPlots.m

```diff
@@ -1,6 +1,8 @@
-function ControlPlots(sysvector, fconv_gpsalt)
+function ControlPlots(sysvector, paramvector, params, fconv_gpsalt)
 % Display the low level controller data.
 
+lines_ = lines(7);
```

Comment:
What is the purpose of this line?

Reply:
For plot colors, see https://github.com/ethz-asl/fw_px4_plottools/blob/12a5cc5c0a3f888fd3b788cf283143601872b3ac/01_draw_functions/01_subfunctions/ControlPlots.m#L89